scripts/common.py

```diff
@@ -150,11 +151,14 @@ def require_cwd_to_be_oppia(allow_deploy_dir=False):
 
 def open_new_tab_in_browser_if_possible(url):
     """Opens the given URL in a new browser tab, if possible."""
-    python_utils.PRINT(
-        'Do you want the url to be opened in the browser? '
-        'Confirm by entering y/ye/yes.')
-    open_new_tab = python_utils.INPUT()
-    if open_new_tab not in ['y', 'ye', 'yes']:
+    if not hasattr(open_new_tab_in_browser_if_possible, 'open_new_tab'):
```

Comment:
In general please don't use hasattr (and this approach seems pretty hacky/unmaintainable anyway). Couldn't you set some kind of local constant/variable in this file?

Reply:
Updated.